patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -266,6 +274,21 @@
 +    }
 +
 +    /**
++     * State-sensitive version of func_176214_u.
++     * <p>
++     * This method is used for:
++     * <br> Checking if an entity should be suffocating
++     * <br> Checking if the suffocation overlay should be rendered in 1st person
++     *
++     * @param state The current state
++     * @return True if the block can suffocate entities that are inside it.
++     */
++    public boolean causesSuffocation(IBlockState state)
```

Comment:
OK, the actual 1.11 methods are `func_191058_s` (`IBlockProperties.isViewBlocking`) and `func_176214_u` (`Block.causesSuffocation`). Mojang didn't update the default implementation though >_>